plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergV2.java

```diff
@@ -113,6 +123,7 @@ protected QueryRunner createQueryRunner()
         return IcebergQueryRunner.builder()
                 .setInitialTables(NATION)
                 .setMetastoreDirectory(metastoreDir)
+                .addAdditionalCatalog(new BlackHolePlugin(), "blackhole", "blackhole")
```

Comment:
`IcebergQueryRunner.addAdditionalCatalog` is redundant, you can achieve the same with
```
queryRunner.installPlugin(new BlackHolePlugin());
queryRunner.createCatalog(...);
```

(currently requires some boilerplate to do robustly
```java
DistributedQueryRunner queryRunner = IcebergQueryRunner.builder()
        .setInitialTables(NATION)
        .setMetastoreDirectory(metastoreDir)
        .build();
try {
    queryRunner.installPlugin(new BlackHolePlugin());
    queryRunner.createCatalog("blackhole", "blackhole");
} catch (Exception e) {
    Closables.closeAllSuppress(e, queryRunner::close);
    throw e;
}
return queryRunner;
```
)